jbpm-test-util/src/main/java/org/jbpm/test/persistence/util/PersistenceUtil.java

```diff
@@ -187,6 +187,15 @@ public static PoolingDataSourceWrapper setupPoolingDataSource(Properties dsProps
         return DataSourceFactory.setupPoolingDataSource(datasourceName, dsProps);
     }
 
+    /**
+     * Sets up a PoolingDataSourceWrapper with the specified datasource.
+     *
+     * @return PoolingDataSourceWrapper that has been set up but _not_ initialized.
+     */
+    public static PoolingDataSourceWrapper setupPoolingDataSource(String datasourceName){
```

Comment:
Maybe just add in the docs that it will use the default `datasource.properties` file.

Reply:
done